variables.tf

```diff
@@ -83,10 +97,10 @@ variable "tagName" {
   description = "Name tag for the servers"
 }
 
-variable "vpc_id" {
-  description = "The AWS VPC ID which you want to deploy your instances"
+variable "user_data" {
+  default = ""
 }
 
-variable "associate_public_ip_address" {
-  default = false
+variable "vpc_id" {
```

Comment:
Totally agree with @tfhartmann . It is not a good sign to expect human-friendly names as input from machines :)